requirements/bddtests.txt

```diff
@@ -42,11 +42,6 @@ h-pyramid-sentry==1.2.3
     # via -r requirements/requirements.txt
 h-vialib==1.0.15
     # via -r requirements/requirements.txt
-httplib2==0.19.0
-    # via
-    #   -r requirements/requirements.txt
-    #   oauth2
-    #   pylti
```

Comment:
Some nice eliminations of dependencies of dependencies. Our whole set of deps has gotten quite a bit smaller throughout the Python 3.6 upgrade process.